functional_tests/scylla_operator/test_functional.py

```diff
@@ -121,3 +121,21 @@ def test_mgmt_backup(tester, db_cluster):
     mgr_task = mgr_cluster.create_backup_task(location_list=[bucket_name, ])
     status = mgr_task.wait_and_get_final_status(timeout=54000, step=5, only_final=True)
     assert TaskStatus.DONE == status
+
+
+def test_listen_address(tester, db_cluster):
+    """
+    Issues: https://github.com/scylladb/scylla-operator/issues/484
+            https://github.com/scylladb/scylla/issues/8381
+    Fix commit: https://github.com/scylladb/scylla-operator/pull/529
+    """
+    all_errors = []
+    for node in db_cluster.nodes:
+        with node.actual_scylla_yaml() as scylla_yaml:
```

Comment:
Currently s-o override this parameter via command line, so we need to handle it properly to:

```

```
```suggestion
        result = node.remoter.run("ps aux | grep docker-entrypoint.py | grep -Po '\-\-listen-address=[^ ]+' | sed -r 's/--listen-address[= ]([^ ]+)/\1/'", ignore_status=True)
        if result.ok:
          if result.stdout != '0.0.0.0':
            all_errors.append(....)
          continue
        
        with node.actual_scylla_yaml() as scylla_yaml:
```